ddtrace/contrib/urllib3/__init__.py

```diff
@@ -0,0 +1,44 @@
+"""
+The ``urllib3`` integration instruments tracing on http calls with optional
+support for distributed tracing across services the client communicates with.
+
+The ``patch`` function must be called before importing and using the library.
```

Comment:
We have a bit of a convention for the documentation that can be copied from the redis/django/grpc integrations